apps/src/lib/node/ledger/ethereum_node/events.rs

```diff
@@ -1,9 +1,9 @@
 pub mod signatures {
     pub const TRANSFER_TO_NAMADA_SIG: &str =
         "TransferToNamada(uint256,(address,uint256,string)[],uint256)";
-    pub const TRANSFER_TO_ETHEREUM_SIG: &str = "TransferToErc(uint256,\
-                                                (address,address,uint256,\
-                                                string,uint256)[],string)";
+    pub const TRANSFER_TO_ETHEREUM_SIG: &str =
+        "TransferToErc(uint256,(address,address,string,uint256,string,\
```

Comment:
the ethereum events are also generated by `ethbridge-rs`. later, we could replace the events here with the generated ones, to avoid having to mess around with type signatures manually